src/components/pages/DetailedProblemView/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { FC, useEffect, useState } from "react";
+import { Link, useParams } from "react-router-dom";
+import { useQuery } from "@apollo/client";
+
+import { GetProblemsQuery, PROBLEMS } from "../ProblemIndex/interface";
+import { ProblemSummaryProps } from "../../molecules/SearchResultTile/interface";
+
+const DetailedProblemView: FC = () => {
+  const params = useParams();
+  const id: any = params.problemId;
+
+  const { data, loading } = useQuery<GetProblemsQuery>(PROBLEMS);
```

Comment:
We can rethink it, when implementing this page precisely, but we need for sure additional call to the API to give us more details about the page, but we should treat it as a separate task, so let's not change the approach here and now. ;)